packages/now-cli/test/dev/integration.js

```diff
@@ -1619,3 +1619,13 @@ test(
     await testPath(200, '/index.css', 'This is index.css');
   })
 );
+
+test(
+  '[vercel dev] Should support `*.go` API serverless functions',
+  testFixtureStdio('go', async testPath => {
+    await testPath(200, `/api`, 'This is the index page');
+    await testPath(200, `/api/another`, 'This is another page');
```

Comment:
Let's also add `await testPath(200, `/api/another.go`, 'This is another page');` just to be sure since the file extension should also work